components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -237,6 +239,13 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
               pattern="http[s]?://.*\..{2,}"
               name="linkedin_url"
             />
+            <TextInput
+              classNames="bg-light-slate-4 text-light-slate-11 font-medium"
+              placeholder="https://discordapp.com/users/832877193112762362"
+              label="Discord URL"
+              pattern="http[s]?://.*\..{2,}"
```

Comment:
Not sure what this is validating. It's not checking for discordapp in the domain name, or the presence of at least 4 numbers in the user id?

Reply:
Oops, just copy/paste that :sweat_smile: It should be good now. The ID should be at least `17` digits because Discord utilizes [Twitter's snowflake format](https://github.com/twitter-archive/snowflake/tree/snowflake-2010) for uniquely identifiable descriptors (IDs)